docs/developer-manual/terms-of-service.md

```diff
@@ -0,0 +1,157 @@
+a result of the underlying data being volatile or otherwise compromised at the
+data source, which could include malfeance or malpractice.
+
+Some of our Services require holding and paying Distributed Ledger Technology (DLT) 
+tokens (also known as "crypto-currency") including ADA and FACT. You hereby represent
+and warrant that you: (1) have the necessary technical expertise and ability to review
+and evaluate the security, integrity and operation of any tokens that you decide to
+acquire, sell or use as a result of using our Services (2) you have the knowledge, 
+experience, understanding, professional advice and information to make your own
+evaluation of the merits, risks and applicable compliance requirements, under Applicable
+Law, of any ADA or FACT tokens that are used as inputs or outputs to Orcfax Services; and 
+(3) know, understand and accept the financial risks associated with your use of
+Orcfax Services and any tokens that are used as inputs or outputs to deliver the Services.
+
+## INDEMNIFICATION
+You will defend, indemnify, and hold harmless Orcfax, our directors, officers,
+employees, attorneys, agents, representatives, suppliers, licensors and contractors,
+including those persons and entities operating software that contributes Oracle data
+to Orcfax (collectively, the "Protected Parties” or individually a "Protected Party")
+from (1) any claim, demand, lawsuit, action, proceeding, investigation, liability, 
+damage, loss, cost or expense, including without limitation reasonable attorneys’ fees,
+arising out of or relating to your use of, or conduct in connection with, 
+Orcfax's Services, including our Oracle data, Archival data, the Website, ADA and FACT
+tokens associated with our or your Distributed Ledger Technology Address; (2) your
+violation of these terms; (3) your violation of Applicable Law or regulations; (4)
+any claims made by or against the Protected Parties by you or other members of the
+organization or entity on whose behalf you may be using the Website or Services; 
+(5) or your infringement or misappropriation of the rights of any other person or entity.
+If you are obligated to indemnify any Protected Party, Orcfax (or, at its discretion,
+the applicable Protected Party) will have the right, in its sole discretion, to
+control any action or proceeding and to determine whether Orcfax wishes to
+settle, and if so, on what terms.
+
+## CHANGES; SUSPENSION; OR TERMINATION OF SERVICES
+Orcfax may, at our sole discretion, from time to time and with or without prior notice to you,
+modify, suspend or disable, temporarily or permanently, our offered Services, in whole or in 
+part, for any reason whatsoever, including, but not limited to, as a result of demands and 
+preferences communicated by users of our Services, a security incident, your violation of these 
+Terms or, in the Orcfax's good faith judgment, such changes, suspension or termination as are
+necessary for the protection of our Services. 
```

Comment:
This is a bit of a word salad from "you violation of these..." but it could be acceptable legalese, but I'm not sure. Might just be worth re-reading and checking for clarity.

Reply:
This is sticking very close to the disclaimer templates we're using. Most of these sentences are not great grammar but are written to be as inclusive or exclusive as possible.